server/data/educationAndWorkPlanClient.test.ts

```diff
@@ -207,4 +208,46 @@ describe('educationAndWorkPlanClient', () => {
       expect(actual).toEqual(expectedActionPlanSummaryListResponse)
     })
   })
+
+  describe('getTimeline', () => {
+    it('should get Timeline', async () => {
+      // Given
+      const prisonNumber = 'A1234BC'
+      const systemToken = 'a-system-token'
+
+      const expectedTimelineResponse = aValidTimeline()
```

Comment:
This should be:
```
const expectedTimelineResponse = aValidTimelineResponse()
```
which means you'll need to write a test data builder for a `TimelineResponse` (keep the test builder you already have for a `Timeline` - we'll use that later. But _this_ test needs a `TimelineResponse`)